diff --git a/node_modules/webtorrent/lib/file-stream.js b/node_modules/webtorrent/lib/file-stream.js
index a3682b1..a517f85 100644
--- a/node_modules/webtorrent/lib/file-stream.js
+++ b/node_modules/webtorrent/lib/file-stream.js
@@ -36,7 +36,7 @@ class FileStream extends stream.Readable {
     this._notifying = false
     this._criticalLength = Math.min((1024 * 1024 / pieceLength) | 0, 2)
 
-    this._torrent.select(this._startPiece, this._endPiece, true, () => {
+    this._torrent.select(this._startPiece, this._endPiece, 100, () => {
       this._notify()
     })
 
